bwa3_comp_ace/CfgVehicles.hpp

```diff
@@ -110,6 +113,21 @@ class CfgVehicles {
         };
     };
 
+    class BWA3_Eagle_Base: Car_F {
+        
+        fuelCapacity = 700 * FUEL_FACTOR;
+        ACE_refuel_fuelCapacity = 180;
+        
+        class Turrets: Turrets {
```

Comment:
Das geht so nicht. Die FLW ist ist nur im mainTurret in class BWA3_Eagle_flw100_base